src/main.cpp

```diff
@@ -2380,7 +2380,7 @@ int main(int argc, char *argv[]) {
               	         if (fld_lr_c[i] > 0.5) {
 		             //Good results with comment below. 
 		             //flensout_f << std::fabs((double)fld_lr[i] / (double)fld_lr_c[i] - index.k);//index.target_lens_[i] - (double)fld_lr[i] / (double)fld_lr_c[i] - k); // take mean of recorded uniquely aligning read lengths 
- 		             flensout_f << std::fabs(index.target_lens_[i] - ((double)fld_lr[i] / (double)fld_lr_c[i]) - index.k);
```

Comment:
Care to elaborate a bit? Ideally in comment, else maybe at least in the commit message?

Reply:
Hi! Based on our analysis for effective length normalization for long reads, the updated effective length provides better results.

Reply:
I still don't follow what you are changing and why but I am not familiar with the code. So if this makes sense to others without further explanation, feel free to ignore my comment.